src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -173,6 +185,18 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
 
   }
 
+  private def toMichelsonScript[T <: MichelsonElement:Parser](json: Any) = {
```

Comment:
When a JSON string cannot be parsed into Michelson, could we store in the database a value such "Unparsable code"?